src/legacy/index.html

```diff
@@ -63,6 +64,8 @@
 <script type="text/javascript" src="https://cdn.ons.gov.uk/vendor/clipboard-js/1.5.9/clipboard.min.js"></script>
 <script type="text/javascript" src="https://cdn.ons.gov.uk/vendor/jquery-tag-it/2.0/tag-it.min.js"></script>
 
+<script src="https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/js/select2.min.js"></script>
```

Comment:
Think it would be better to pull the minified library then add it to the `src/legacy/js/third-party` directory. 

We don't control this CDN, so if it goes down or files go missing there isn't anything we can do and all our Florence users would be impacted. As this is Legacy Florence.

Reply:
Yeah that makes sense and thought would be the case